src/it/ajc/verify.groovy

```diff
@@ -30,3 +30,5 @@
 
 assert !new File(basedir, 'target/classes/jcabi-ajc.log').exists()
 assert new File(basedir, 'target/jcabi-ajc.log').exists()
+assert new File(basedir, 'target/classes/com/jcabi/foo/Sample.class').exists()
+assert new File(basedir, 'target/unwoven/com/jcabi/foo/Sample.class').exists()
```

Comment:
@amihaiemil do you see any possibility to check that this file contents is different than the above one as it stays unwoven?